testlapack/general.go

```diff
@@ -17,8 +17,14 @@ import (
 	"github.com/gonum/lapack"
 )
 
-// dlamchE is the machine epsilon. For IEEE this is 2^-53.
-const dlamchE = 1.0 / (1 << 53)
+const (
+	// dlamchE is the machine epsilon. For IEEE this is 2^-53.
+	dlamchE = 1.0 / (1 << 53)
+	dlamchP = 2 * dlamchE
+	dlamchB = 2
+)
+
+var dlamchS = math.Nextafter((4 / math.MaxFloat64), 0)
```

Comment:
I changed it to const using the expression you suggested. Later we can do the same change in native.